src/commands/create-project.js

```diff
@@ -16,33 +16,46 @@ const builder = yargs => {
 		.option("org", {
 			alias: "o",
 			describe: "Organization",
-			demandOption: true,
-			type: "string"
+			type: "boolean"
 		})
-		.option("name", {
-			alias: "n",
-			describe: "Project name",
-			demandOption: true,
-			type: "string"
+		.option("repo", {
+			alias: "r",
+			describe: "Repository",
+			type: "boolean"
+		})
+		.option("user", {
+			alias: "u",
+			describe: "GitHub username",
+			type: "boolean"
+		})
+		.conflicts({
+			org: ["repo", "user"],
+			repo: ["org", "user"],
+			user: ["org", "repo"]
+		})
+		.middleware((argv, yargs) => {
```

Comment:
From a yargs documentation dive 🏊‍♂️, I think it'd be better to use the yargs `check` function rather than middleware.

http://yargs.js.org/docs/#api-checkfn-globaltrue

> Check that certain conditions are met in the provided arguments.
> 
> `fn` is called with two arguments, the parsed `argv` hash and an array of options and their aliases.
> 
> If `fn` throws or returns a non-truthy value, show the thrown error, usage information, and exit.

So this handles a lot of what is being manually defined in the middleware.

Reply:
Might be useful to reference, https://stackoverflow.com/a/54868875.

> yargs demand one of builder option

Reply:
👍 Done! Thanks for the feedback, I've pushed the changes and I think I've addressed all of your points 😄